src/App.js

```diff
@@ -1,7 +1,23 @@
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import LandingPage from "./components/LandingPage/LandingPage";
+import ContactPage from "./components/ContactPage/ContactPage";
+import MenuPage from "./components/MenuPage/MenuPage";
+import AdminPanel from "./components/AdminPage/AdminPanel/AdminPanel";
+import AdminHome from "./components/AdminPage/AdminHome/AdminHome";
+
 function App() {
   return (
-    <div className="App">
-    </div>
+    <Router>
+      <div className="App">
+        <Routes>
+          <Route path="/" element={<LandingPage />} />
+          <Route path="/contact" element={<ContactPage />} />
+          <Route path="/menu" element={<MenuPage />} />
+          <Route path="/admin" element={<AdminPanel />} />
+          <Route path="/admin/home" element={<AdminHome />} />
```

Comment:
This route should be protected. Nobody except the logged in user should be allowed to enter here. Do you remember about this?

Reply:
Yes, I remember :)